src/helper.js

```diff
@@ -0,0 +1,40 @@
+//@ts-check
+
+import fs from "fs";
+import path from "path";
+
+export function checkFileIsExisting(filePath) {
+  if (fs.existsSync(filePath)) {
+    return true;
+  } else {
+    console.log(
+      `report does not exist, are you sure there is a json report in ${filePath} path?`
+    );
+    return false;
+  }
+}
+
+export function parseJsonFile(filePath) {
+  if (checkFileIsExisting(filePath)) {
+    return JSON.parse(fs.readFileSync(filePath, "utf8"));
+  }
+}
+
+export function writeJsonFile(data, outputDir, fileName) {
+  let _path = path.resolve(outputDir, fileName);
+  fs.writeFileSync(_path, JSON.stringify(data));
+}
+
+export function orderBasedOnBrowserDuration(specs, browser) {
+  return specs.sort(function (a, b) {
+    let aDuration = a.data.find((item) => item.browser === browser).duration;
+    let bDuration = b.data.find((item) => item.browser === browser).duration;
```

Comment:
What if data won't be found? Then `a.data.find((item) => item.browser === browser).duration` will be `undefined.duration` which will end in "Uncaught TypeError: Cannot read property 'duration' of undefined"

Reply:
here I'm trying to order a pre-defined json file which has been generated by the orchestrator itself.